src/Services/ConvertToQtiService.php

```diff
@@ -2,36 +2,41 @@
 
 namespace LearnosityQti\Services;
 
-use LearnosityQti\AppContainer;
+use DOMDocument;
+use DOMElement;
+use Exception;
 use LearnosityQti\Converter;
 use LearnosityQti\Domain\JobDataTrait;
-use LearnosityQti\Exceptions\MappingException;
+use LearnosityQti\Processors\IMSCP\Entities\File;
+use LearnosityQti\Processors\IMSCP\Entities\ImsManifestMetadata;
+use LearnosityQti\Processors\IMSCP\Entities\Manifest;
+use LearnosityQti\Processors\IMSCP\Entities\Resource;
 use LearnosityQti\Processors\QtiV2\Out\Constants as LearnosityExportConstant;
-use LearnosityQti\Utils\AssetsFixer;
-use LearnosityQti\Utils\AssumptionHandler;
-use LearnosityQti\Utils\CheckValidQti;
-use LearnosityQti\Utils\ResponseProcessingHandler;
 use LearnosityQti\Utils\General\FileSystemHelper;
-use LearnosityQti\Utils\General\StringHelper;
+use LearnosityQti\Utils\UuidUtil;
+use LearnosityQti\Utils\General\CopyDirectoreyHelper;
+use RecursiveIteratorIterator;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\Iterator\RecursiveDirectoryIterator;
 use Symfony\Component\Finder\SplFileInfo;
-use qtism\data\AssessmentItem;
-use qtism\data\content\ItemBody;
-use qtism\data\storage\xml\XmlDocument;
+use ZipArchive;
 
 class ConvertToQtiService
 {
     use JobDataTrait;
 
     const RESOURCE_TYPE_ITEM = 'imsqti_item_xmlv2p1';
     const INFO_OUTPUT_PREFIX = '';
-    const CONVERT_LOG_FILENAME = 'convert:to:qti.log';
+    const CONVERT_LOG_FILENAME = 'converttoqti.log';
```

Comment:
For consistency can we use the same format as here (with dashes)?
https://github.com/Learnosity/learnosity-qti/pull/47/files#diff-6f0e68dc90c0979d95f5acaf95f64b4eR29

Reply:
@ttton  This will work in Unix/linux environment but this will not work windows enviorement as colon (:) is reserved special character in windows .
Do we still need to change it ?.

Reply:
We're not using colons in the other file any more, we're using dashes instead.

Reply:
I'm requesting we change it to the new format (might be clearer to look at the blob instead: https://github.com/Learnosity/learnosity-qti/blob/1c2e59fd302cf46fd7ac3a67f826865d1afe50d2/src/Services/ConvertToLearnosityService.php#L29)

Reply:
Done